src/strigo/scripts/configs/__init__.py

```diff
@@ -26,13 +24,14 @@ class Script():
 
     @staticmethod
     def from_dict(script_type: ScriptType, d: Dict[str, Any]) -> Script:
+        d['env'] = {k: (v if isinstance(v, str) else json.dumps(v, separators=(',', ':'))) for k, v in d.get('env', {}).items()}
```

Comment:
not blocking, but I discourage the use of one-lettered variable names, even inside comprehension expressions. It makes code review harder for others or your future self.

"Code is read more often than written" is one of Python's motto and the advice goes beyond writing in Python only.

Reply:
Here `k` and `v` are as classic as `i` for a loop index. Also changing for `key` and `value` doesn't bring much more meaning.

Reply:
they may be classic names, using them is still considered as a bad practice (http://pylint-messages.wikidot.com/messages:c0103) :smiley: The same applies for the `d` dictionary in the signature.

Some suggestions to help people understanding the code:
- rename `d` into `script_options` or `vm_initialisation_options`
- `k`: `vm_initialisation_key`
- `v`: `vm_initialisation_value`

It may look overkill when you work often on this library (thank you, by the way!), but it really helps reviewing and understanding the code